demo/admin/src/common/ComponentDemo.tsx

```diff
@@ -183,32 +169,24 @@ export function ComponentDemo(): React.ReactElement {
                                 }
                             >
-                                {(props) => (
-                                    <FinalFormSelect {...props}>
-                                        <MenuItem value="Option 1">1</MenuItem>
-                                        <MenuItem value="Option 2">2</MenuItem>
-                                        <MenuItem value="Option 3">3</MenuItem>
-                                    </FinalFormSelect>
-                                )}
-                            </Field>
-
-                            <Field name="select-custom" label="Custom select" fullWidth>
-                                {(props) => (
-                                    <FinalFormSelect {...props}>
-                                        <MenuItem value="Option 1">
-                                            <CustomSelectItem icon={<Snips />} primary="Option 1" secondary="Secondary text" />
-                                        </MenuItem>
-                                        <MenuItem value="Option 2">
-                                            <CustomSelectItem icon={<Snips />} primary="Option 2" secondary="Secondary text" />
-                                        </MenuItem>
-                                        <MenuItem value="Option 3">
-                                            <CustomSelectItem icon={<Snips />} primary="Option 3" secondary="Secondary text" />
-                                        </MenuItem>
-                                    </FinalFormSelect>
-                                )}
-                            </Field>
-
-                            <Field name="textArea" label="Text Area" component={FinalFormInput} multiline minRows={3} fullWidth />
+                                <MenuItem value="Option 1">1</MenuItem>
+                                <MenuItem value="Option 2">2</MenuItem>
+                                <MenuItem value="Option 3">3</MenuItem>
+                            </SelectField>
+
+                            <SelectField name="select-custom" label="Custom select" fullWidth>
+                                <MenuItem value="Option 1">
+                                    <CustomSelectItem icon={<Snips />} primary="Option 1" secondary="Secondary text" />
+                                </MenuItem>
+                                <MenuItem value="Option 2">
+                                    <CustomSelectItem icon={<Snips />} primary="Option 2" secondary="Secondary text" />
+                                </MenuItem>
+                                <MenuItem value="Option 3">
+                                    <CustomSelectItem icon={<Snips />} primary="Option 3" secondary="Secondary text" />
+                                </MenuItem>
+                            </SelectField>
+
+                            <TextAreaField name="textArea" label="Text Area" multiline minRows={3} fullWidth />
```

Comment:
`multiline` (and possibly `minRows`) should not be necessary, when using `TextAreaField`